test/gtest/ucs/test_type.cc

```diff
@@ -48,6 +49,42 @@ UCS_TEST_F(test_type, status) {
     EXPECT_NE(UCS_OK, UCS_PTR_STATUS(ptr));
 }
 
+/* Represents latency (in ns) */
+UCS_FP8_DECLARE_TYPE(LATENCY, UCS_BIT(7), UCS_BIT(20))
```

Comment:
in UCT/UCP code latency is represented in seconds. can we make it work with fp8 ?

Reply:
In current implementation it won't work out of the box, because I made it under the given assumptions of positive number, "large" (>1) numbers.

Will make adjustments.